client/src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from "react";
+import { OutlinedInput, InputLabel, InputAdornment } from "@material-ui/core";
+import LocationOnIcon from "@material-ui/icons/LocationOn";
+import LocationOffIcon from "@material-ui/icons/LocationOff";
+import { isDebug } from "src/config";
+
+function SearchBar({
```

Comment:
This is not a general text based search bar but location based, hence this should be named something more specific.
```suggestion
function LocationSearchBar({
```

Reply:
Intended this to be a search bar with icon clicking functionality, renamed into IconSearchBar, icons are now passed as props